cli/hzn.go

```diff
@@ -90,6 +90,12 @@ Environment Variables:
 	cliutils.Opts.Verbose = app.Flag("verbose", msgPrinter.Sprintf("Verbose output.")).Short('v').Bool()
 	cliutils.Opts.IsDryRun = app.Flag("dry-run", msgPrinter.Sprintf("When calling the Horizon or Exchange API, do GETs, but don't do PUTs, POSTs, or DELETEs.")).Bool()
 
+	envCmd := app.Command("env", msgPrinter.Sprintf("Show the Horizon Environment Variables."))
+	envOrg := envCmd.Flag("org", msgPrinter.Sprintf("The Horizon organization ID. If not specified, HZN_ORG_ID will be used as a default.")).Short('o').String()
```

Comment:
I played around with this command a bit and I dont think we need any of the switches.